### DISCUSSION TITLE
Ai 女團翻車mv

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#271] [General]
越來越多娛樂公司推出ai女團，eternity 是韓國推出的第一個全ai人物的偶像團體，但釋出的mv中可以發現ai人物在唱跳時的臉部表情及肢體非常詭異。

https://udn.com/news/story/7086/5371752

**Comment by zixunn** [comment]
不知道為甚麼影片越看越好笑

**Comment by 107602505** [comment]
有點不太懂為何要將AI應用在偶像團體上面，那個影片中每個人的臉都好像P上去的超級詭異XD

**Comment by chun1203** [comment]
整個看起來超詭異 然後歌名還叫I’m Real 格外諷刺XD

**Comment by 109802503gloria** [comment]
看到韓國做的AI主播表情蠻自然的，如果可以把技術放到這裡應該會好很多

**Comment by cody910120** [comment]
女團信仰Blackpink

**Comment by mesakiyuta** [comment]
眼神特別詭異，死魚眼的感覺，整體來說都太僵硬了

**Comment by yt7l0** [comment]
表情好空洞無神，而且沒在看鏡頭XD
可以期待他改良一下

**Comment by vvvioletta** [comment]
每個人的臉都好像⋯沒有記憶點XD

**Comment by YuHengChen54** [comment]
真的好詭異，這技術還要再多加強阿...

**Comment by Joan0518** [comment]
笑死每個人都長好像，完全是美顏開太多的那種感覺，臉都糊糊的很假

**Comment by zhiron544** [comment]
我覺得ai人物的偶像團體很有創意，但是我還是喜歡真人的偶像多一些。我覺得真人的偶像有更多的個性和魅力，也能夠和粉絲有更多的互動和溝通。

**Comment by sunset513** [comment]
我覺得 ai 有它的優勢，條件許可下，它能夠近乎無限制的產出作品。但我還是更喜歡真實的人，而非很像真人的人，畢竟真人更能帶給我真實的感覺。真人帶有一點個人的獨特魅力、或是先天的不完美，這種貼近人性的偶像才是更吸引我的。

**Comment by LinChloe** [comment]
不喜歡AI 女團，韓國每個女團都更有特色，性格分明，每個人都有不同魅力，還可以讓我吃午餐時看團綜。

**Comment by happy246810** [comment]
我個人還是傾向於真人女團，因為真人女團中的每一個人都有不同的性格，而且如果有演唱會的話，還可以與粉絲互動與溝通，而AI女團雖然能不斷的產生新作品，但沒有與歌迷互動，缺乏真實感。

**Comment by cassy0201** [comment]
對比之下，韓國真實的女團還是有存在的必要，畢竟能真實的與粉絲互動、交流，它們的存在也會為粉絲帶來情緒價值

**Comment by 710388** [comment]
還是真實的比較好，畢竟真實的根本不是虛擬可以比的，虛擬的太假了還是覺得不夠擬真

**Comment by NcuJerryKao** [comment]
Eternity 作為韓國首個全AI人物的偶像團體，的確引起了廣泛關注。針對你提到的臉部表情及肢體動作詭異的問題，以下是一些建設性的建議：

技術改進：

目前AI技術在模仿真人臉部表情和肢體動作方面仍有改進空間，建議娛樂公司加強與AI技術公司的合作，進一步提升AI人物的自然度[[1](https://www.thenewslens.com/article/152287)]。
混合技術使用：

可考慮使用真人替身進行動作捕捉，再通過AI技術進行臉部和動作的細節調整，這樣能提高MV的真實感和自然度[[6](https://today.line.me/hk/v2/article/qoP3YMK)]。
粉絲互動：

增加AI偶像與粉絲的互動，例如虛擬演唱會、線上粉絲見面會等，讓粉絲更能接受和喜愛這些AI偶像。
多樣化內容：

除了MV外，可以嘗試推出更多元的內容，例如綜藝節目、生活短片等，讓AI偶像展現更多面貌，增加親和力和真實感。
透過這些改進措施，AI女團Eternity有望在未來贏得更多粉絲的喜愛與支持。

**Comment by tshsu04224** [comment]
去年有推出了另一個虛擬團體叫做mave，我覺得整體MV,舞蹈,表演等等都比當時的eternity進步了許多，也更加自然，重點是歌還是不錯聽的，現在有越來越多這種虛擬團體出現，未來會成為趨勢也說不定，附上MV有興趣的朋友可以去欣賞看看
[https://youtu.be/1wGOHbcQKIc?si=9f2QTaNnGy6tRUma](https://youtu.be/1wGOHbcQKIc?si=9f2QTaNnGy6tRUma)

**Comment by 155Julia** [comment]
當我第一次觀看Eternity這個全AI偶像團體的MV時，我的感受相當複雜。這些AI人物的外貌雖然相當逼真，但在唱跳時的臉部表情和肢體動作卻顯得非常詭異，甚至有些不協調。我認為，雖然AI技術在娛樂領域的應用前景廣闊，但目前仍存在許多技術上的瓶頸需要突破。

**Comment by poiu6fi** [comment]
現在很多AI的影片還是會看出很多奇怪的地方，像是臉部，不過之後科技進步不知道會不會連我們都分不出來了，這樣感覺有點可怕

**Comment by JustAHedgehog** [comment]
雖然目前已經沒辦法看到版主提供的影片，但我個人發現可以從這件事看出世人對虛擬偶像與真人偶像的觀看標準不同，真人偶像會針對他們的長相、身材變化、興趣、私生活等等個人取向問題，而AI偶像則是會要求有沒有像真人，有沒有哪裡破圖、不像真人等等是否為人的話題。

**Comment by Nana1218** [comment]
相較真人女團，AI在臉部表情或是肢體流暢度上與真人偶像還是存在著一定的差距，但我覺得這在未來，差距是有可能被縮小的，科技持續進步，若是配合效能好的顯示器，AI和真人偶像在這些部分的表現可能會平分秋色。
